proto/desmos/subspaces/v1beta1/msgs.proto

```diff
@@ -0,0 +1,153 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// Msg defines subspaces Msg service.
+service Msg {
+
+  // CreateSubspace defines the method to create a subspace
+  rpc CreateSubspace(MsgCreateSubspace) returns (MsgCreateSubspaceResponse);
+
+  // TransferOwnership defines the method to transfer a subspace ownership
+  rpc TransferOwnership(MsgTransferOwnership) returns (MsgTransferOwnershipResponse);
+
+  // AddAdmin defines the method to add a new admin to the subspace
+  rpc AddAdmin(MsgAddAdmin) returns (MsgAddAdminResponse);
+
+  // RemoveAdmin defines the method to remove an admin from a specific subspace
+  rpc RemoveAdmin(MsgRemoveAdmin) returns (MsgRemoveAdminResponse);
+
+  // EnableUserPosts defines the method to let user posts inside a specific subspace
+  rpc EnableUserPosts(MsgEnableUserPosts) returns (MsgEnableUserPostsResponse);
+
+  // DisableUserPosts defines the method to block user posts inside a specific subspace
+  rpc DisableUserPosts(MsgDisableUserPosts) returns (MsgDisableUserPostsResponse);
```

Comment:
Instead of `MsgEnableUserPosts` and `MsgDisableUserPosts` I think it would be much clearer if we used more widely-used terms such as `MsgRegisterUser` and `MsgBlockUser` that might make it easier for external developers to immediately understand what those are used for